dev/com.ibm.ws.microprofile.faulttolerance.3.0.metrics/bnd.bnd

```diff
@@ -0,0 +1,44 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.3.0.metrics
+Bundle-SymbolicName: com.ibm.ws.microprofile.faulttolerance.3.0.metrics
+
+Bundle-Description:MicroProfile Fault Tolerance 3.0 metrics integration, version ${bVersion}
+
+WS-TraceGroup: FAULTTOLERANCE
+
+-dsannotations: \
+        com.ibm.ws.microprofile.faulttolerance.metrics.integration.MetricRecorderProvider30Impl
+
+-buildpath: \
+        com.ibm.ws.logging;version=latest, \
+        com.ibm.ws.logging.core;version=latest, \
+        com.ibm.ws.kernel.service;version=latest,\
+        org.eclipse.osgi;version=latest, \
+        com.ibm.websphere.org.osgi.core;version=latest, \
+        com.ibm.websphere.org.osgi.service.component;version=latest, \
+        com.ibm.wsspi.org.osgi.service.component.annotations;version=latest, \
+        com.ibm.ws.microprofile.faulttolerance;version=latest, \
+        com.ibm.ws.microprofile.faulttolerance.spi;version=latest, \
+        com.ibm.websphere.org.eclipse.microprofile.metrics.2.3;version=latest, \
+        com.ibm.websphere.org.eclipse.microprofile.config.1.3;version=latest,\
```

Comment:
What is the minimum version of MP Config that FT 3.0 will require? The feature file above says it tolerates 1.1-1.4 ... in which case, this build path should use the 1.1 API? I realise that is subject to change because it will probably also have to support MP Config 2.0.

Reply:
We decided we'd leave this as-is for now, but will update it when we have the features available for MP 4.0